elide-graphql/src/test/java/com/yahoo/elide/graphql/GraphQLEndpointTest.java

```diff
@@ -800,12 +786,7 @@ public void testMultipleRoot() throws JSONException {
                                 "author",
                                 selections(
                                         field("id"),
-                                        field("name")
```

Comment:
This is no longer testing multiple roots.  Let's rename the method or remove the test.

Reply:
This one still tests two root level query, one author and one book without alias.